ac/ac-autocode/src/Dashboard.jsx

```diff
@@ -3,23 +3,152 @@
 import React from 'react';
 
 import { type LogDBT } from 'frog-utils';
+import {
+  VictoryBar,
+  VictoryChart,
+  VictoryAxis,
+  VictoryStack,
+  VictoryLegend,
+  VictoryLabel
+} from 'victory';
 
-const Viewer = (props: Object) => (
-  <div>
-    <h1>A nice Dashboard</h1>
-    <p>Data collected:</p>
-    <pre>{JSON.stringify(props.data, null, 2)}</pre>
-  </div>
-);
+const Viewer = (props: Object) => {
+  // props = {users, instances, data, config}
+  const chartData = value =>
+    props.config.tests.map((test, index) => ({
+      x: 'test ' + index,
+      y: props.data.tests[index] ? props.data.tests[index][value] : 0
+    }));
+
+  const debugData = () => {
+    const o = Object.keys(props.users).reduce((acc, userId) => {
```

Comment:
The viewer receives as props the list of users and instances. I think the dashboards should be based on instances. Logs also have the instanceId to be used in the merge function